samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1736,6 +1736,38 @@
           }
         ]
       }
+    },
+    "AthenaQueryPolicy": {
+      "Description": "Gives permissions to execute Athena queries",
+      "Parameters": {
+        "WorkGroupName": {
+          "Description": "Name of the Athena Workgroup"
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "athena:GetWorkGroup",
+              "athena:GetQueryExecution",
+              "athena:StartQueryExecution",
+              "athena:StopQueryExecution",
+              "athena:GetQueryResults"
```

Comment:
verfied these policies from link: https://docs.aws.amazon.com/athena/latest/APIReference/API_Operations.html

Reply:
@nheijmans looking at this AWS docs there are more permissions required for accessing a specified athena workgroup - https://docs.aws.amazon.com/athena/latest/ug/example-policies-workgroup.html
Could you add those as well?

Reply:
@praneetap Yep I'll add them as well!